lib/board.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../lib/player.rb'
+
+class Board
+  attr_accessor :game_board, :slots_taken
```

Comment:
Remember that having `attr_accessor` allows your variable to be read and written from anywhere in your application. Make sure you actually need them or change them to `attr_reader` whenever you can.